### DISCUSSION TITLE
Does superset support mobile device?

### DISCUSSION THREAD

**Discussion by xlhtc007** [apache/superset#13816] [Ideas]
I am considering using superset as BI tool to our team and we have the mobile device requirement. I have not found the clear instruction that superset support mobile device.

**Comment by srinify** [comment]
Hi there, Superset doesn't work on mobile devices at the moment.

- Mobile app: no mobile app for Superset
- Mobile web support: some features *may* work on mobile but Superset isn't tested regularly for smaller viewports

**Comment by junlincc** [comment]
Thanks Srini! 

@xlhtc007 Are you looking for chart/dashboard view only on mobile? or more like interactive? Neither is in our near-term roadmap. It isn't hard to implement view-only mobile i believe, and some forks have done it in the past. It will be really great to have Superset work on mobile; please let us know if you would love to work on it :)

**Comment by curiousus** [comment]
Hi @xlhtc007 , we use a workaround in our company so that dashboard can be (relatively) comfortably used on mobile devices:

- make all charts taller and wider (at least 6 lanes for Big Numbers and Filterboxes, all 12 lanes for other chart types)
- use CSS to make the text content larger

It requires a lot of fine tuning and some trial-and-error with CSS, but it is doable.
